tests/unit/imgix-component.spec.ts

```diff
@@ -1,54 +1,74 @@
+import VueImgix from '@/plugins/vue-imgix/index';
+import { IxImg } from '@/plugins/vue-imgix/ix-img';
+import { IxPicture } from '@/plugins/vue-imgix/ix-picture';
+import { IxSource } from '@/plugins/vue-imgix/ix-source';
+import { render } from '@testing-library/vue';
+import { config, mount } from '@vue/test-utils';
 import { createApp } from 'vue';
-import VueImgix, { IxImg } from '@/plugins/vue-imgix';
-import '@testing-library/jest-dom';
-import { render, screen } from '@testing-library/vue';
 import _App from '../../src/App.vue';
 import {
   expectElementToHaveFixedSrcAndSrcSet,
   expectElementToHaveFluidSrcAndSrcSet,
 } from '../helpers/url-assert';
-
-const App = createApp(_App);
+/**
+ * Why register the plugin and each individual component globally?
+ *
+ * It's a limitation with  @vue/test-utils at the moment. The
+ * `config.global.plugins` option is supposed to register plugins globally,
+ * but it doesn't seem to work when tests are run in parallel.
+ *
+ * The only reliable way to register plugins components globally is to register
+ * them individually as `config.global.components` as well as
+ * `config.global.plugins`.
+ */
+config.global.plugins = [[VueImgix, { domain: 'assets.imgix.net' }]];
+config.global.components = {
+  IxImg,
+  IxPicture,
+  IxSource,
+};
 
 describe('imgix component', () => {
-  beforeAll(() => {
-    App.use(VueImgix, {
-      domain: 'assets.imgix.net',
-    });
-  });
   it('an img should be rendered', () => {
-    render(IxImg, {
+    const wrapper = mount(IxImg, {
       props: {
         src: 'examples/pione.jpg',
-        // TODO(luis): remove this when we have a better way of testing
         ['data-testid']: 'img-rendering',
       },
     });
 
-    expect(screen.getByTestId('img-rendering'));
+    const el = wrapper.find('img[data-testid="img-rendering"]').html();
```

Comment:
Btw, this `data-testid` stuff was added because that's what `@testing-library` requires. If you think the tests would be cleaner without it, feel free to remove them. Up to you.